certbot/log.py

```diff
@@ -309,14 +309,11 @@ def except_hook(exc_type, exc_value, trace, debug, log_path):
             """ Match Group #1: acme-v01.api.letsencrypt.org"""
             """ Match Group #2: [Errno 65]"""
             """ Match Group #3: No route to host"""
-            err_re = re.search(r"host='(\S*)'.*(\[Errno \d+\]\ )(.*)(',\)\))", str(exc_value))
-            if err_re:
-                err_msg = "Can't connect to "+err_re.group(1)+\
-                          "\n"+err_re.group(2)+\
-                          "- "+err_re.group(3)
-                print(err_msg, file=sys.stderr)
-            else:
-                traceback.print_exception(exc_type, exc_value, None)
+            print (exc_value, file=sys.stderr)
+            match = re.match(r".*host='(\S*)'.*url\: (\/\w*).*(\[Errno \d+\])((\s*\w*)*)", str(exc_value))
```

Comment:
I'd still like to see a comment with a full example of a real error message that we're matching against.